CHAOS-3393 Single-click onboarding (#6350)
harness/developer-hub: docs/chaos-engineering/get-started/onboarding/guided-onboarding.md

---
sidebar_position: 2
title: Guided onboarding
description: Set up infrastructure with a single click and execute chaos experiments.
---

This guide guides you through setting up the chaos infrastructure, executing chaos experiments, and viewing the resilience dashboard with single clicks.

## Before you begin

* [Overview of HCE onboarding](/docs/chaos-engineering/get-started/onboarding/hce-onboarding.md)

### Execute chaos experiments

> 1. Navigate to the left nav bar and click **Overview**. Click **Select a Cluster** to choose a cluster so that you can execute chaos experiments.

Reviewer: ```suggestion
1. In the left menu, select **Overview** and then **Select a Cluster** where you can execute chaos experiments.
```

Reviewer: You might also consider indenting the graphics so they align with the first character in the bullet point and shrinking them down a little.  I insert graphics like this, makes it easy to import/resize in one command: 

<DocImage path={require('./static/custom-image-shared-path.png')} width="50%" height="50%" title="Add shared path for scan results" />
